client/app/bundles/course/group/pages/GroupShow/GroupManager/GroupUserManagerList.jsx

```diff
@@ -107,49 +163,41 @@ const GroupUserManagerListItem = ({
 
       <ListItemText primaryTypographyProps={{ style: styles.listItemTextSize }}>
         {user.name}
+        {user.isPhantom ? <GhostIcon /> : ''}
+        {otherGroups ? ` (also a member of${otherGroups})` : ''}
       </ListItemText>
     </div>
 
     {showDropdown ? (
-      <div style={styles.listItemWithDropdown}>
-        <Select
-          onChange={(event) => onChangeDropdown(event.target.value, user)}
-          onClick={() => {}}
-          style={styles.listItemTextSize}
-          value={user.groupRole}
-          variant="standard"
-        >
-          <MenuItem style={styles.listItemTextSize} value="normal">
-            <FormattedMessage {...translations.normal} />
-          </MenuItem>
-          <MenuItem style={styles.listItemTextSize} value="manager">
-            <FormattedMessage {...translations.manager} />
-          </MenuItem>
-        </Select>
-      </div>
+      <GroupUserManagerListItemChoice
+        onChangeDropdown={onChangeDropdown}
+        user={user}
+      />
     ) : null}
   </ListItem>
 );
 
 GroupUserManagerListItem.propTypes = {
   user: memberShape.isRequired,
   colour: PropTypes.object,
+  otherGroups: PropTypes.arrayOf(PropTypes.string),
   onCheck: PropTypes.func.isRequired,
-  showDropdown: PropTypes.bool,
   onChangeDropdown: PropTypes.func,
+  showDropdown: PropTypes.bool,
   isChecked: PropTypes.bool,
 };
 
 const GroupUserManagerList = ({
   students = [],
   staff = [],
+  memberOtherGroups = {},
   onCheck,
   colourMap,
   showDropdown = false,
   onChangeDropdown,
   isChecked = false,
 }) => {
-  const renderUsersListItems = (users, title) => (
+  const renderUsersListItems = (users, members, title) => (
```

Comment:
Is this supposed to be `memberOtherGroups`?

Reply:
this is true.. I just deleted the members variable since it's not used anywhere inside the function itself